docs/reference/remap/literals/object.cue

```diff
@@ -12,9 +12,8 @@ remap: literals: object: {
 		ordering: {
 			title: "Ordering"
 			description: """
-				Objects are ordered alphabetically by the key in ascending order. Therefore, operations, such as
-				encoding into JSON, will produce a string with keys that are alphabetically ordered in ascending
-				fashion.
+				Objects are ordered alphabetically by the key in ascending order. Therefore, operations like
```

Comment:
```suggestion
				Object fields are ordered alphabetically by the key in ascending order. Therefore, operations like
```

Or perhaps it's `Object rows`? The Objects aren't ordered, it's their fields that are..